core-bundle/src/Resources/views/Backend/be_undo_label.html.twig

```diff
@@ -0,0 +1,41 @@
+{% trans_default_domain 'contao_default' %}
+
+<div class="tl_undo_header">
+    <div class="tl_undo_header_item">
+        <span class="tl_undo_header_label">
+            <span class="date">{{ row.tstamp|date(dateFormat) }}</span>
+            <span class="time">{{ row.tstamp|date(timeFormat) }}</span>
+        </span>
+    </div>
+    <div class="tl_undo_header_item">
+        <span class="tl_undo_header_label">{{ 'tl_undo.pid.0'|trans({}, 'contao_tl_undo') }}</span>
+        <strong>{% if user %}{{ user.username }}{% else %}{{ row.pid }}{% endif %}</strong>
+    </div>
+    <div class="tl_undo_header_item">
+        <span class="tl_undo_header_label">{{ 'tl_undo.fromTable.0'|trans({}, 'contao_tl_undo') }}</span>
+        <strong>{{ translatedFromTable }}</strong>
+    </div>
+    <div class="tl_undo_header_item">
+        <span class="tl_undo_header_label">ID</span> {{ originalRow.id }}
```

Comment:
`{{ originalRow.id }}` is the only string that is now wrapped in a `<span>` or `<strong>` tag. Intended?

Reply:
Yes, it is not necessary. No styling needed ATM.